media/css/firefox/whatsnew/whatsnew-90.scss

```diff
@@ -0,0 +1,225 @@
+    text-align: center;
+    max-width: $content-sm;
+
+    @media #{$mq-md} {
+        @include bidi(((text-align, left, right),));
+    }
+}
+
+.mzp-c-picto-image {
+    align-items: flex-end;
+    display: flex;
+    height: 250px;
+    justify-content: center;
+    margin: 0 auto $spacing-lg;
+    max-width: $content-xs;
+    min-height: $layout-md;
+}
+
+.wnp-content-footer {
+    text-align: center;
+
+    .wnp-footer-title {
+        @include text-title-md;
+        margin-bottom: $layout-md;
+    }
+}
+
+
+//* -------------------------------------------------------------------------- */
+// Utilities (link to release notes)
+.c-utilities {
+    @include text-body-sm;
+    max-width: $content-md;
+    padding-bottom: $layout-xl;
+    text-align: center;
+}
+
+
+//* -------------------------------------------------------------------------- */
+// Footer overrides
+.mzp-c-footer.mzp-t-firefox {
+    background: $color-white;
+    color: $color-ink-80;
+
+    a:link,
+    a:visited {
+        color: $color-ink-80;
+    }
+
+    a:hover,
+    a:focus,
+    a:active {
+        color: $color-ink-90;
+    }
+
+    .mzp-c-footer-sections {
+        border-top: 1px solid $color-light-gray-30;
+        border-bottom: 1px solid $color-light-gray-30;
+        padding: $layout-md 0;
+    }
+}
+
+.mzp-c-footer-links-social li {
+    a.twitter {
+        background-image: url('/media/protocol/img/icons/social/twitter/black.svg');
+    }
+
+    a.instagram {
+        background-image: url('/media/protocol/img/icons/social/instagram/black.svg');
+    }
+
+    a.youtube {
+        background-image: url('/media/protocol/img/icons/social/youtube/black.svg');
+    }
+}
+
+.mzp-c-footer-primary-logo a {
+    background-image: url('/media/protocol/img/logos/mozilla/logo-word-hor.svg');
+}
+
+
+//* -------------------------------------------------------------------------- */
+// For dark mode
+@media (prefers-color-scheme: dark) {
+    .content-wrapper {
+        background: $color-dark-gray-60;
+        color: $color-white;
+    }
+
+    .wnp-main-image {
+        @include border-box;
+        background: transparent url('/media/protocol/img/logos/pocket/logo-word-hor-white.svg') center top no-repeat;
+        @include background-size(220px, auto);
+        display: inline-block;
+        height: 60px;
+        overflow: hidden;
+        padding-top: 60px;
+    }
+
+    .wnp-main-tagline .inline-icon {
+        @include border-box;
+        background: transparent url('/media/img/firefox/whatsnew/whatsnew90/pocket-outline-white.svg') center top no-repeat;
+        @include background-size(1em, auto);
+        display: inline-block;
+        height: 1em;
+        overflow: hidden;
+        padding-top: 1em;
+    }
+
+    .wnp-main-title,
+    .wnp-footer-title,
+    .mzp-c-picto-heading {
+        color: get-theme('title-text-color-inverse');
+    }
+
+    .c-utilities,
+    .c-sub-cta {
+        @include white-links;
+    }
+
+    .mzp-c-footer.mzp-t-firefox {
```

Comment:
This doesn't match the markup of the footer in use on this page so dark mode does not get a dark footer.